cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -89,18 +89,16 @@
                m-list__horizontal
                m-social-media_icons">
 
-        {% set facebook_account = 'CFPBespanol' if language == 'es' else 'cfpb' %}
         {% set facebook_info = {
             'name': 'Facebook',
-            'homepage': 'https://facebook.com/' ~ facebook_account,
+            'homepage': _( 'https://www.facebook.com/CFPB' ),
```

Comment:
I've changed this from the naked facebook.com domain to the www.facebook.com subdomain to be more consistent in how we link to FB.